subcommand/injector/flags.go

```diff
@@ -56,6 +56,9 @@ type Specification struct {
 	// TLSAutoHosts is the AGENT_INJECT_TLS_AUTO_HOSTS environment variable.
 	TLSAutoHosts string `envconfig:"tls_auto_hosts"`
 
+	// TLSCACertFile is the AGENT_INJECT_TLS_CA_CERT_FILE environment variable.
+	TLSCACertFile string `envconfig:"tls_ca_cert_file"`
```

Comment:
We'll probably want to add a test case for AGENT_INJECT_TLS_CA_CERT_FILE in [TestCommandEnvs()](https://github.com/kondotak/vault-k8s/blob/a160ece8060796d04732cb640cc5d393b4f7c05f/subcommand/injector/flags_test.go#L110) as well.

Reply:
okay, added test case, easy to add!